reference/vm-tools/qvm-copy-to-vm.md

```diff
@@ -14,15 +14,18 @@ qvm-copy-to-vm
 NAME
 ----
 
-qvm-copy-to-vm - copy specified files to specified destination VM
+qvm-copy - copy specified files to destination VM (chosen afterward in dom0 qubes.Filecopy popup dialog)
+
+deprecated: qvm-copy-to-vm - copy specified files to specified destination VM
+qvm-copy-to-vm/qvm-move-to-vm tools are deprecated 2017-01, use qvm-copy/qvm-move to avoid typing target qube name twice
```

Comment:
qvm-copy-to-vm is deprecated only in Qubes 4.0. In (still supported) 3.2 it is the right to use.

Reply:
ah, sorry about that, I don't have access to a machine running 3.2 right now.  Let's just leave the documentation alone then for now, users can keep using qvm-copy-to-vm in 4.0 without penalty.  Unless there's a wish to have specific sections for Qubes 4.0 in the documentation... maybe better once 4.0 is finalized.  Thanks!

Reply:
just noticed this - https://www.qubes-os.org/doc/doc-guidelines/#version-specific-documentation - but qvm-copy is so minor I won't bother for now.